pallets/loans/src/rate.rs

```diff
@@ -227,7 +227,9 @@ mod tests {
 
     #[test]
     fn get_supply_rate_works() {
-        let borrow_rate = APR::from(Rate::saturating_from_rational(2, 100 * BLOCK_PER_YEAR));
+        // DOT's block per year
+        let block_per_year: u128 = 5256000;
```

Comment:
Does not use `T::BlockPerYear::get()` here?

Reply:
I did this because it seems we don't have a runtime here

Reply:
Oh yes, it is.